ArduCopter/Parameters.cpp

```diff
@@ -1208,6 +1208,13 @@ const AP_Param::GroupInfo ParametersG2::var_info2[] = {
     // @User: Advanced
     AP_GROUPINFO("SURFTRAK_TC", 5, ParametersG2, surftrak_tc, 1.0),
 
+    // @Param: TKOFF_TH_MAX
+    // @DisplayName: Maximum throttle during take-off ramp up
+    // @Description: Maximum throttle allowed before controllers assume the aircraft is airborn druing the takeoff process.
```

Comment:
typo "druing" -> "during"
I also think some wording like "You should reduce this if your takeoff is too agressive."